go.mod

```diff
@@ -33,10 +35,12 @@ require (
 	github.com/gosuri/uitable v0.0.1
 	github.com/hashicorp/go-immutable-radix v1.0.0 // indirect
 	github.com/hashicorp/go-msgpack v0.5.5
+	github.com/hashicorp/go-retryablehttp v0.6.8
```

Comment:
Is this actually required? 

We're very conscious about what libraries we pull into Juju. We require each one to be vetted to ensure that they don't overreach and we have to end up supporting this for the maintenance window of a release (can be quite a long time!).

Reply:
This is the http client library used by github.com/packethost/packngo.  If this is a blocker we could look at removing it upstream, letting the user provide the retryable client on their own (the features of this client are used in the Terraform and other integrations).

Reply:
What @displague said we should remove this in upstream.

Reply:
https://github.com/packethost/packngo/pull/222 this going to be removed once this is merged upstream.